spring-cloud-gcp-autoconfigure/src/main/java/com/google/cloud/spring/autoconfigure/trace/pubsub/TracingSubscriberFactory.java

```diff
@@ -48,7 +48,7 @@ public PullRequest createPullRequest(String subscriptionName, Integer maxMessage
 	}
 
 	@Override
-	public SubscriberStub createSubscriberStub() {
-		return pubSubTracing.subscriberStub(delegate.createSubscriberStub());
+	public SubscriberStub createSubscriberStub(String subscriptionName) {
```

Comment:
Wouldn't this be considered a breaking change? Can we keep both versions of the method, and deprecate one?

Reply:
That is a good catch! This is indeed a breaking change and you're right that it will be safer to deprecate this and add a new one. I will make this change in a follow-up PR.